docs/distributing-policies/secure-supply-chain.md

```diff
@@ -442,33 +512,30 @@ anyOf: # at least `anyOf.minimumMatches` are required to match
     key: .... # mandatory
 ```
 
-The configuration is composed of two sections:
+</details>
+
+The configuration contains the two highlighted sections:
+
+- `allOf`: The policy is trusted only if all signature requirements in this section are valid.
 
-- `anyOf`:  The policy will be trusted if the `minimumMatches` criterion is fulfilled.
-In the above example, the `minimumMatches` field has been assigned a value 2.
-This means that at least two of the signature requirements listed **need to be fulfilled**
-for the policy to be trusted. The default value that the `minimumMatches` field assumes is `1`.
-- `allOf`: All the signature requirements defined in this section must be valid
-so that the policy can be trusted.
+- `anyOf`:  The policy is trusted if the `minimumMatches` criterion is met.
+Above, the `minimumMatches` field is 2.
+So, at least two of the signature requirements must be fulfilled.
+The default value for `minimumMatches` field is `1`.
 
-Each of these sections can contain one or more signature requirements.
-Users can also define both sections in one file as shown above. In this situations, all
-the signatures requirements from the `allOf` **AND** a minimum number of matches
-from the `anyOf` section as described in the `minimumMatches` field must be satisfied.
+All the signatures requirements from `allOf` **and** a minimum number from `anyOf` must be met.
 
 ### Signature validation
 
-The users can validate different signatures in the `anyOf` and `allOf` sections.
+<!--TODO:What are we trying to say here?-->
```

Comment:
Certain organization have compliance rules that require a container image/policy to be signed by at least X/Y trusted users or by all the trusted users. This is done to reduce the impact of having one of the trusted users being compromised by an attacker

Reply:
Thanks. I've had another go at that bit. Needs another review iteration though.